### ISSUE TITLE
 Unable to load localization data for Microsoft.AspNetCore.Components.IHandleEvent

### ISSUE THREAD

**Issue by dodyg** [xaviersolau/BlazorJsonLocalization#52]
I got the localization working on my Blazor WASM app (8 Preview 4) but I keep finding this error in the log. What could be the problem?

```
SoloX.BlazorJsonLocalization.Core.Impl.JsonStringLocalizerFactory[14]
      Unable to load localization data for Microsoft.AspNetCore.Components.IHandleEvent in assembly Microsoft.AspNetCore.Components, Version=8.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60 with culture en-US
```

**Comment by xaviersolau** [comment]
Is your component implementing IHandleEvent ?

**Comment by dodyg** [comment]
![image](https://github.com/xaviersolau/BlazorJsonLocalization/assets/1761249/32194c60-c593-4191-94cc-fad11bf19abc)

**Comment by xaviersolau** [comment]
Basically, it is trying to load resources through yours razor component inheritance hierarchy. In our case IHandleEvent and probably IHandleAfterRender, IComponent, ComponentBase. This is resulting in this log message telling you that it can’t find any resources for those classes.

It should be a warning message instead of an error message. I will change this in next release.

And I will probably add an option to disable logs.

**Comment by dodyg** [comment]
Wouldn't it be better/faster if we can configure skipping certain classes?

**Comment by xaviersolau** [comment]
Yes, this is a good idea. I will add an optional filter.

**Comment by xaviersolau** [comment]
Available in version 2.0.0
